packages/swapper/src/swappers/zrx/utils/test-data/setupSwapQuote.ts

```diff
@@ -0,0 +1,48 @@
+import { ContractTypes, NetworkTypes, ChainTypes, Asset } from '@shapeshiftoss/asset-service'
+import { DEFAULT_SLIPPAGE } from '../constants'
+
+export const setupQuote = () => {
+  const sellAsset = {
+    name: 'Fox',
+    chain: ChainTypes.Ethereum,
+    network: NetworkTypes.MAINNET,
+    precision: 18,
+    tokenId: '0xc770eefad204b5180df6a14ee197d99d808ee52d',
+    contractType: ContractTypes.ERC20,
+    color: '#FFFFFF',
+    secondaryColor: '#FFFFFF',
+    icon: 'https://assets.coincap.io/assets/icons/fox@2x.png',
+    sendSupport: true,
+    receiveSupport: true,
+    symbol: 'FOX'
+  } as unknown as Asset
```

Comment:
add some todos to remove these when we have the `lib/types` package with unified `ChainTypes` and `ChainIdentifier`